deployer/conf/fluent.conf

```diff
@@ -0,0 +1,33 @@
+# This file is the fluentd configuration entrypoint. Edit with care.
+
+@include configs.d/openshift/system.conf
+
+# In each section below, pre- and post- includes don't point at anything by default;
+# they are there to enable admins to add conf as needed.
+
+## sources
+## ordered so that syslog always runs last...
+@include configs.d/openshift/input-pre-*.conf
+@include configs.d/dynamic/input-docker-*.conf
+@include configs.d/dynamic/input-syslog-*.conf
+@include configs.d/openshift/input-post-*.conf
+##
+
+<label @INGRESS>
+## filters
+  @include configs.d/openshift/filter-pre-*.conf
+  @include configs.d/openshift/filter-k8s-meta.conf
+  @include configs.d/openshift/filter-kibana-transform.conf
+  @include configs.d/openshift/filter-k8s-flatten-hash.conf
+  @include configs.d/openshift/filter-k8s-record-transform.conf
+  @include configs.d/openshift/filter-syslog-record-transform.conf
+  @include configs.d/openshift/filter-post-*.conf
+##
+
+## matches
+  @include configs.d/openshift/output-pre-*.conf
+  @include configs.d/openshift/output-operations.conf
+  @include configs.d/openshift/output-applications.conf
+  @include configs.d/openshift/output-post-*.conf
```

Comment:
Hadn't thought of that. Seems like we might as well just remove it then?

On Tue, Jul 12, 2016 at 4:15 PM, Eric Wolinetz <notifications@github.com>
wrote:

> In deployer/conf/fluent.conf
> <https://github.com/openshift/origin-aggregated-logging/pull/195#discussion_r70513406>
> :
>
> > +<label @INGRESS>
> > +## filters
> > +  @include configs.d/openshift/filter-pre-*.conf
> > +  @include configs.d/openshift/filter-k8s-meta.conf
> > +  @include configs.d/openshift/filter-kibana-transform.conf
> > +  @include configs.d/openshift/filter-k8s-flatten-hash.conf
> > +  @include configs.d/openshift/filter-k8s-record-transform.conf
> > +  @include configs.d/openshift/filter-syslog-record-transform.conf
> > +  @include configs.d/openshift/filter-post-*.conf
> > +##
> > +
> > +## matches
> > +  @include configs.d/openshift/output-pre-*.conf
> > +  @include configs.d/openshift/output-operations.conf
> > +  @include configs.d/openshift/output-applications.conf
> > +  @include configs.d/openshift/output-post-*.conf
>
> important to note that since output-applications.conf does a match on **
> we won't ever get to this output-post-*
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/195/files/c79a4f8ff336fad4187dcb823b73638724df6f84#r70513406>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABz-vpoNKCI1kZ2psJ1g77MLhOS7nqPks5qU_X5gaJpZM4JHWV8>
> .
>